plugins/dicom_viewer/plugin.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "DICOM Viewer",
+    "description": "View DICOM images in the browser",
+    "version": "0.1.0",
+    "npm": {
+        "dependencies": {
+            "daikon": "^1.2.15",
+            "kw-web-suite": "^2.0.1",
+            "vtk.js": "github:kitware/vtk-js"
```

Comment:
Is it possible to pin to a release version of vtk.js so that this plugin won't break unexpectedly?